src/prometheus_alert_rules/metrics_alert_rules.yaml

```diff
@@ -21,7 +25,11 @@ groups:
           severity: warning
         annotations:
           summary: MySQL too many connections (> 90%) (instance {{ $labels.instance }})
-          description: "More than 90% of MySQL connections are in use on {{ $labels.instance }}\n  VALUE = {{ $value }}\n  LABELS = {{ $labels }}"
+          description: |
```

Comment:
For this I would change the summary since `too much` connections would be 100%.
Suggestion: "MySQL is approaching max connections count"